incubating/aws-s3/step.yaml

```diff
@@ -0,0 +1,101 @@
+kind: step-type
+version: '1.0'
+metadata:
+  name: aws-s3
+  version: 1.0.0
+  isPublic: true
+  description: Integration that uploads files to S3
+  sources:
+    - https://github.com/codefresh-io/steps/tree/master/incubating/aws-s3
+  stage: incubating
+  maintainers:
+    - name: Luke Goodfellow
+    - email: luke.goodfellow@codefresh.io
+  categories:
+    - utilities
+  official: true
+  tags: []
+  icon:
+    type: svg
+    url: https://raw.githubusercontent.com/codefresh-io/steps/aws-s3/incubating/aws-cdk/images/s3-logo.svg
```

Comment:
I think you're missing the branch and the aws-s3 should be below incubating
For reference, the one for ServiceNow plugin
https://raw.githubusercontent.com/codefresh-io/steps/master/incubating/service-now/images/servicenow-2.png

so  
https://raw.githubusercontent.com/codefresh-io/steps/master/incubating/aws-s3/images/s3-logo.svg

Reply:
You are Correct I will make the correction.